itests/hive-blobstore/src/test/results/clientpositive/write_final_output_blobstore.q.out

```diff
@@ -306,7 +306,7 @@ POSTHOOK: type: QUERY
 OPTIMIZED SQL: SELECT `key`
 FROM `default`.`hdfs_table`
 GROUP BY `key`
-ORDER BY `key`
+ORDER BY `key` NULLS LAST
```

Comment:
In L1449 of CalcitePlanner.java, we have this block:
```
SqlDialect dialect = new HiveSqlDialect(SqlDialect.EMPTY_CONTEXT
        .withDatabaseProduct(SqlDialect.DatabaseProduct.HIVE)
        .withDatabaseMajorVersion(4) // TODO: should not be hardcoded
        .withDatabaseMinorVersion(0)
        .withIdentifierQuoteString("`")
        .withNullCollation(NullCollation.LOW))
```
Can you set ```NullCollation``` value depending on HiveConf value? This will avoid a bunch of changes in the q files that contain the optimized SQL such as the one above, since when default NULLS ordering is used, it will be not be included in the ORDER BY clause.

Reply:
I made NullCollation depending on HiveConf. Thanks.